Project.toml

```diff
@@ -16,7 +16,7 @@ TextAnalysis = "a2db99b7-8b79-58f8-94bf-bbc811eef33d"
 CorpusLoaders = "0.3"
 MLJModelInterface = "1.3"
 ScientificTypes = "2.2.2"
```

Comment:
This will hopefully fix 1.7 CI.

```
ScientificTypes = "2.2.2, 3"
```

Reply:
Should we commit this change and merge this PR?